gdcmodels/__init__.py

```diff
@@ -35,7 +49,10 @@ def get_es_models():
                 es_models[es_index][es_type] = {
                         '_mapping': load_yaml(pj(es_model_dir, es_index, f))
                     }
-
+                defs_name = pj(es_model_dir, es_index,es_type+".".join(['_definitions', 'yaml']))
+                if isfile(defs_name):
+                    definitions = load_definitions(defs_name)
+                    es_models[es_index][es_type]['_mapping'].update(definitions)
```

Comment:
if `$ref` is in the file, you'll want to clean it up as well.

Reply:
that's correct thanks